avocado/core/runners/sysinfo.py

```diff
@@ -0,0 +1,311 @@
+import logging
+import multiprocessing
+import os
+import time
+import traceback
+from abc import ABC, abstractmethod
+
+from avocado.core import nrunner
+from avocado.core.runners.utils import messages
+from avocado.core.settings import settings
+from avocado.utils import astring, process, software_manager
+
+log = logging.getLogger("avocado.sysinfo")
+
+
+class _Collectible(ABC):
```

Comment:
Any reason for  making this and  next classes privates?   If for hiding during import, maybe we  could use ` __all__` here.

Reply:
I think it is more related to the code organization. I don't expect anything from this file to be imported by other files.

Reply:
Right, then we could use `__all__` for  that.

Reply:
The reason is exactly what @willianrampazzo says. I will use `__all__`.